torch/_export/serde/schema.py

```diff
@@ -250,6 +250,7 @@ class GraphModule:
     # TODO(zhxchen17) Merge call_spec into call graph.
     call_spec: CallSpec
     module_call_graph: List[ModuleCallEntry]
+    constants: str
```

Comment:
@angelayi could we make it a bit more structured to be Dict[str, bytes]? At least backend don't need to implement unpacking for dictionaries now.

Reply:
Also I think constants belongs to Graph, sorry I wasn't 100% clear back then. The reason is cond() branches could also have constants and you sort of wanna register them separately?

Reply:
Oh right..sorry. I think it is a Dict[str, bytes] 😅